api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -34,6 +34,16 @@ service Disperser {
   // For an example usage, see how our disperser_client makes a call to this endpoint to populate its local accountant struct:
   // https://github.com/Layr-Labs/eigenda/blob/6059c6a068298d11c41e50f5bcd208d0da44906a/api/clients/v2/disperser_client.go#L298
   rpc GetPaymentState(GetPaymentStateRequest) returns (GetPaymentStateReply) {}
+
+  // GetQuorumSpecificPaymentState is a utility method to get the payment state of a given account, at a given disperser.
+  // EigenDA's payment system for v2 is currently centralized, meaning that each disperser does its own accounting.
+  // A client wanting to disperse a blob would thus need to synchronize its local accounting state with that of the disperser.
+  // That typically only needs to be done once, and the state can be updated locally as the client disperses blobs.
+  // The accounting rules are simple and can be updated locally, but periodic checks with the disperser can't hurt.
+  //
+  // For an example usage, see how our disperser_client makes a call to this endpoint to populate its local accountant struct:
+  // https://github.com/Layr-Labs/eigenda/blob/6059c6a068298d11c41e50f5bcd208d0da44906a/api/clients/v2/disperser_client.go#L298
+  rpc GetQuorumSpecificPaymentState(GetQuorumSpecificPaymentStateRequest) returns (GetQuorumSpecificPaymentStateReply) {}
```

Comment:
Think renaming to `GetPaymentStateQuorumSpecific` might be cleanest. Any IDE (or maybe even grpcurl reflection) that will list rpc endpoints alphabetically will show GetPaymentState and GetPaymentStateQuorumSpecific next to each other, which isn't true for GetQuorumSpecificPaymentState.